single_stage_detector/ssd/run_and_time.sh

```diff
@@ -36,8 +36,10 @@ BATCHSIZE=${BATCHSIZE:-2}
 EVALBATCHSIZE=${EVALBATCHSIZE:-${BATCHSIZE}}
 NUMEPOCHS=${NUMEPOCHS:-30}
 LOG_INTERVAL=${LOG_INTERVAL:-20}
+DATASET=${DATASET:-"openimages-mlperf"}
```

Comment:
MLPerf requires us to use the same dataset as in the reference to ensure results are comparable across submissions. So we should not make it a changeable parameter